com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Matrix/Matrix2Node.cs

```diff
@@ -20,5 +20,10 @@ internal class Matrix2Node : IStandardNode
             { "Tooltip", "creates a static 2x2 matrix" },
             { "Parameters.Out.Tooltip", "a 2x2 matrix" }
         };
+
+        public static Dictionary<string, float> UIHints => new()
+        {
+            { "Preview.Exists", 0 }
```

Comment:
A thing that would be nice is the ability to have these UI Hint keys defined in a central place as compile time constants, available to both the GraphDelta and GraphUI assemblies, meaning that if we ever wanted to change the name of a key to a UIHint, say, we'd only have to change it one place as opposed to across all the nodes that use it